x-pack/packages/ml/data_frame_analytics_utils/src/constants.ts

```diff
@@ -39,6 +39,7 @@ export const JOB_MAP_NODE_TYPES = {
   TRANSFORM: 'transform',
   INDEX: 'index',
   TRAINED_MODEL: 'trainedModel',
+  INGEST_PIPELINE: 'ingestPipeline',
```

Comment:
There are no actions available in the flyout for this type of node. Is it possible to hide the Node actions button in this case:

<img width="198" alt="image" src="https://github.com/elastic/kibana/assets/7405507/be2c8439-f37c-46a9-bfc1-5b6796ce86c5">

Reply:
I've updated so that:
 1) If there's no action available, it will not show this button
<img width="1792" alt="Screen Shot 2023-07-27 at 15 08 13" src="https://github.com/elastic/kibana/assets/43350163/2280030e-50df-45c3-a8d9-dbfd20dcaaaa">
 2) If user has Kibana access, there will be a link to navigate to the ingest pipeline management page

https://github.com/elastic/kibana/assets/43350163/a246cb03-ef26-4d36-af0f-185b8037c696